External/FEXCore/Source/Interface/Core/CPUID.cpp

```diff
@@ -25,89 +28,321 @@ CPUIDEmu::FunctionResults CPUIDEmu::Function_0h() {
 CPUIDEmu::FunctionResults CPUIDEmu::Function_01h() {
   CPUIDEmu::FunctionResults Res{};
 
-  Res.Res[0] = 0 | // Stepping
+  Res.eax = 0 | // Stepping
     (0 << 4) | // Model
     (0 << 8) | // Family ID
     (0 << 12) | // Processor type
     (0 << 16) | // Extended model ID
     (0 << 20); // Extended family ID
-  Res.Res[1] = 0 | // Brand index
+  Res.ebx = 0 | // Brand index
     (8 << 8) | // Cache line size in bytes
     (8 << 16) | // Number of addressable IDs for the logical cores in the physical CPU
     (0 << 24); // Local APIC ID
-  Res.Res[2] = ~0U; // Let's say we support every feature for fun
-  Res.Res[3] = ~0U; // Let's say we support every feature for fun
-
-  Res.Res[3]  &= ~(
-      (1 << 1) |  // Remove CLMUL
-      (3 << 26) | // Let's say that XSAVE isn't enabled by the OS. Prevents glibc from using XSAVE/XGETBV
-      (1 << 9)  | // Remove SSSE3
-      (1 << 19) | // Remove SSE4.1
-      (1 << 20) | // Remove SSE4.2
-      (1 << 25) | // Remove AES
-      (1 << 28) | // Remove AVX
-      (1 << 30)   // Remove RDRAND
-      );
+  Res.ecx =
+    (1 <<  0) | // SSE3
+    (0 <<  1) | // PCLMULQDQ
+    (1 <<  2) | // DS area supports 64bit layout
+    (1 <<  3) | // MWait
+    (1 <<  4) | // DS-CPL
+    (0 <<  5) | // VMX
+    (0 <<  6) | // SMX
+    (0 <<  7) | // Intel SpeedStep
+    (0 <<  8) | // Thermal Monitor 2
+    (0 <<  9) | // SSSE3
+    (0 << 10) | // L1 context ID
+    (0 << 11) | // Silicon debug
+    (1 << 12) | // FMA
```

Comment:
1. Should be labled FMA3
2. I don't think we support FMA3.

Reply:
Fixed